pkg/apis/operator/v1alpha1/tektonpipeline_validation.go

```diff
@@ -59,11 +62,29 @@ func (p *PipelineProperties) validate(path string) (errs *apis.FieldError) {
 		}
 	}
 
+	if p.MaxResultSize != nil {
+		if *p.MaxResultSize >= 1572864 {
```

Comment:
@piyush-garg can you add some reference about `1572864`, from where did we get this

Reply:
This field has limit of value equal to 1.5MB as mentioned here https://github.com/tektoncd/pipeline/blob/main/docs/additional-configs.md#enabling-larger-results-using-sidecar-logs

And I took the value from here https://github.com/tektoncd/pipeline/blob/main/pkg/apis/config/feature_flags.go#L346

Reply:
Thanks @piyush-garg 
It will be nicer, if we add those supporting docs as comment on top of the block.